packages/client/components/NewMeetingCarousel.tsx

```diff
@@ -139,18 +138,17 @@ const NewMeetingCarousel = (props: Props) => {
         modules={[Mousewheel, FreeMode, Keyboard]}
         mousewheel={true}
         slidesOffsetBefore={24}
-        slidesOffsetAfter={16}
-        slideToClickedSlide={true}
-        spaceBetween={16}
+        slidesOffsetAfter={24}
+        centeredSlides
+        centeredSlidesBounds
+        slideToClickedSlide
+        spaceBetween={24}
         threshold={10}
         keyboard={true}
-        slidesPerView={1.5}
+        slidesPerView={1.99}
```

Comment:
there's some magic here. if we `2` slides are visible, then it won't `slideToClickedSlide` for the 2nd slide moving forward or slide 1 when moving backwards :shrug: